ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ViewVulnReportPage.tsx

```diff
@@ -27,8 +27,8 @@ import {
 } from '@patternfly/react-core/deprecated';
 import { CaretDownIcon } from '@patternfly/react-icons';
 
-import { vulnerabilityReportPath } from 'Containers/Vulnerabilities/VulnerablityReporting/pathsForVulnerabilityReporting';
-import { vulnerabilityReportsPath } from 'routePaths';
+import { vulnerabilityConfigurationReportPath } from 'Containers/Vulnerabilities/VulnerablityReporting/pathsForVulnerabilityReporting';
+import { vulnerabilityConfigurationReportsPath } from 'routePaths';
```

Comment:
Nit: the almost identical naming of these values made me carefully look to see what the difference was. I could see these getting mixed up in implementation and without an automated test it could easily go unnoticed.

Reply:
That's true. We could differentiate them by an extra word like this:
```
vulnerabilityConfigurationReportDetailsPath
vulnerabilityConfigurationReportsListPath
```
If it doesn't look too long

Reply:
^ Actually, we don't want to modify `vulnerabilityConfigurationReportsPath` since it follows the format we see for other paths. I do see some paths use a `_BasePath` format. Maybe we could use that? For now, I just changed `vulnerabilityConfigurationReportPath` to `vulnerabilityConfigurationReportDetailsPath`. I think that should be good enough.